client/app/src/main/java/com/microsoft/research/karya/ui/splashScreen/SplashScreenFragment.kt

```diff
@@ -54,11 +51,6 @@ class SplashScreenFragment : Fragment(R.layout.fragment_splash_screen) {
     requireActivity().finish()
   }
 
-  private fun navigateToUserAuthScreen() {
-    navController.navigate(R.id.action_splashScreenFragment_to_registration_navigation)
-    requireActivity().finish()
-  }
-
   private fun navigateToAccessCodeScreen() {
     navController.navigate(R.id.action_splashScreenFragment_to_access_code_nav_graph)
     requireActivity().finish()
```

Comment:
not needed since you are already finishing activity on line 51

Reply:
Nice catch, I'll update it.

Reply:
Nvm, it is not really an issue, we are calling `finish()` on each method.